test/src/tests/generated/body-complex/models.go

```diff
@@ -1103,6 +1221,160 @@ type Siamese struct {
 	Breed             *string `json:"breed,omitempty"`
 }
 
+// SmartSalmon
+type SmartSalmon struct {
```

Comment:
Did these came from the new version of the test server ? I don't think we have proper tests for this. Maybe create an issue to add tests for polymorphic chain scenarios

Reply:
Correct.  Note that we are missing a lot of the Azure-specific tests.

Reply:
This is tracked in issue https://github.com/Azure/autorest/issues/1582